docker/deploy/config/sakai.properties

```diff
@@ -0,0 +1,5345 @@
+## sakai.properties - the default edition

```

Comment:
It would be nice if we didn't include this file here as its a duplicate of config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties
the maven resources plugin can copy this see http://maven.apache.org/plugins/maven-resources-plugin/copy-resources-mojo.html

Reply:
Agreed, I reduced it to the minimal configuration needed for the example stack.